client/me/get-apps/index.jsx

```diff
@@ -1,82 +1,104 @@
 /**
  * External dependencies
  */
-var React = require( 'react' );
-
+import React, { PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
 /**
  * Internal dependencies
  */
-var MeSidebarNavigation = require( 'me/sidebar-navigation' ),
-	Main = require( 'components/main' ),
-	Button = require( 'components/button' ),
-	Card = require( 'components/card' );
-
-module.exports = React.createClass( {
-
-	displayName: 'GetApps',
-
-	render: function() {
-		return (
-			<Main className="get-apps">
-				<MeSidebarNavigation />
-
-				<h1 className="get-apps__title">{ this.translate( 'Get Apps' ) }</h1>
-				<p className="get-apps__subheading">{ this.translate( 'Manage all your WordPress.com and Jetpack-enabled sites in one place. On your desktop, or on the go.' ) }</p>
+import MeSidebarNavigation from 'me/sidebar-navigation';
+import Main from 'components/main';
+import Button from 'components/button';
+import Card from 'components/card';
+import SectionHeader from 'components/section-header';
 
-				<h2 className="form-section-heading">{ this.translate( 'Desktop apps' ) }</h2>
-				<Card className="get-apps__desktop">
+const defaultProps = {
+  displayName: 'GetApps',
+};
+const propTypes = {
+  translate: PropTypes.func.isRequired,
+};
+const GetApps = ( { translate } ) => (
+  <Main className="get-apps">
+    <MeSidebarNavigation />
+    <SectionHeader label={ translate( 'Desktop apps' ) } />
+    <Card className="get-apps__desktop">
 
-					<section className="get-apps__app">
-						<div className="get-apps__app-icon">
-							<img src="/calypso/images/me/get-apps-osx.svg" alt={ this.translate( 'Mac' ) } width="96" height="96" />
-						</div>
-						<p><strong>{ this.translate( 'Mac' ) }</strong><br />
-						{ this.translate( 'Requires 10.11 or newer' ) }</p>
-						<Button href="https://apps.wordpress.com/d/osx?ref=getapps">{ this.translate( 'Download' ) }</Button>
-					</section>
+      <section className="get-apps__app">
+        <div className="get-apps__app-icon">
+          <img src="/calypso/images/me/get-apps-osx.svg" alt={ translate( 'Mac' ) } width="96" height="96" />
+        </div>
+        <p><strong>{ translate( 'Mac' ) }</strong><br />
+        { translate( 'Requires 10.11 or newer' ) }</p>
+        <Button href="https://apps.wordpress.com/d/osx?ref=getapps">{ translate( 'Download' ) }</Button>
```

Comment:
I realize that this was here already but it could be good to update this translation with context as `Download` could mean different things. We have `context` we can pass along to indicate parts of speech.

```js
translate( 'Download', { context: 'Verb' } )
```